packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -173,6 +177,11 @@ abstract class CachedArtifact {
   }
 
   Future<void> update() async {
+    // Clear any existing artifact files.
+    if (file.parent.existsSync()) {
```

Comment:
Do we want to delete the entire parent directory? This is assuming that each artifact will be cached in its own nested directory (which I'm not sure is a valid assumption) 🤷 ?

Reply:
Updated to only delete the artifact and stamp file.